### DISCUSSION TITLE
Browser-based Ruby IDE

### DISCUSSION THREAD

**Discussion by sebnozzi** [ruby/ruby.wasm#218] [Show and tell]
I'm working on a browser-based Ruby IDE (unpublished).

![Screenshot of Ruby IDE](https://github.com/ruby/ruby.wasm/assets/1067921/6c088e76-97fb-4cd4-ad31-800142207f21)

The project actually uses Opal for the Ruby evaluation, but I have been evaluating Ruby/Wasm on a separate branch.

Performance-wise results have been good. But I'm running into trouble trying to implement a step-by-step debugger using async-evaluation and TracePoint line-handlers.

**Comment by neskuk** [comment]
hoi seb
ich kann mich in keinen meiner zwei discord-konten einliggen. bitte melde dich via
neskuk affenschwanz gmail dot com
danke im voraus
